src/Symfony/Bundle/FrameworkBundle/Command/CachePoolDeleteCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Delete an item from a cachhe pool.
+ *
+ * @author Pierre du Plessis <pdples@gmail.com>
+ */
+final class CachePoolDeleteCommand extends Command
+{
+    protected static $defaultName = 'cache:pool:delete';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDefinition(array(
+                new InputArgument('pool', InputArgument::REQUIRED, 'The cache pool to delete an item from'),
+                new InputArgument('key', InputArgument::REQUIRED, 'The cache key to delete from the pool'),
+            ))
+            ->setDescription('Deletes an item from a cache pool')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> deletes an item from a given cache pool.
+
+    %command.full_name% <pool> <key>
+EOF
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+        $pool = $input->getArgument('pool');
+        $key = $input->getArgument('key');
+        $cachePool = $this->getApplication()->getKernel()->getContainer()->get($pool);
```

Comment:
This means the pool has to be public.
What about going through the Psr6CacheClearer instead?